TestAssets/TestProjects/AppWithDirectDependencyWithOutputName/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilationOptions": {
+    "emitEntryPoint": true,
+    "preserveCompilationContext": true
+  },
+  "dependencies": {
+    "ToolWithOutputName": {
```

Comment:
Do we also want a test with this tool being in the "tools" section instead of dependencies?